spec/models/test_track/session_spec.rb

```diff
@@ -158,6 +148,50 @@ def current_clown; end
             )
           end
         end
+
+        context 'when analytics client is not default' do
```

Comment:
Ahh, I see the tests are here. I think we can move some of these tests to SafeWrapper since they're mostly testing its behavior.